src/Collector/ProfilePlugin.php

```diff
@@ -21,31 +20,13 @@ class ProfilePlugin implements Plugin
 {
     use Plugin\VersionBridgePlugin;
 
-    /**
-     * @var Plugin
-     */
-    private $plugin;
-
-    /**
-     * @var Collector
-     */
-    private $collector;
-
-    /**
-     * @var Formatter
-     */
-    private $formatter;
-
-    public function __construct(Plugin $plugin, Collector $collector, Formatter $formatter)
+    public function __construct(private Plugin $plugin, private Collector $collector, private Formatter $formatter)
```

Comment:
```suggestion
    public function __construct(
        private Plugin $plugin,
        private Collector $collector,
        private Formatter $formatter
    )
```

Reply:
++

Reply:
you did not actually commit this one ;-)

Reply:
Sorry, replaced with regexp mentioned "readonly"